components/molecules/CardRepoList/card-repo-list.tsx

```diff
@@ -89,9 +87,9 @@ const CardRepoList = ({
                   ) : (
                     ""
                   )}
-                </div>
+                </button>
               );
-            })}
+            })}{" "}
```

Comment:
Refactor (blocking): A blank space text node got added that shouldn't bet there.

```suggestion
            })}
```